README.md

```diff
@@ -61,11 +75,11 @@ These great products are built on Semantic-UI-React. Add yours [here][22].
 - http://www.brewhousesolutions.com
 - https://www.lifebot.fr
 
-## Voice Your Opinion :speech_balloon:
+## Voice Your Opinion
```

Comment:
Some headers had emoji and others did not.  Standardized README headers to not have any emoji.